package.json

```diff
@@ -185,13 +185,24 @@
     "pnpm": "^8.9.0"
   },
   "dependencies": {
-    "nx": "^15.9.2",
+    "nx": "^16.10.0",
     "tslib": "^2.4.0"
   },
   "pnpm": {
     "overrides": {
       "proxy-agent": "^6.3.0",
-      "nth-check": "^2.1.1"
+      "nth-check": "^2.1.1",
+      "braces@<2.3.1": "^2.3.1",
+      "nanoid@>=3.0.0 <3.1.31": "^3.1.31",
+      "file-type@>=13.0.0 <16.5.4": "^16.5.4",
+      "glob-parent@<5.1.2": "^5.1.2",
+      "xml2js@<0.5.0": "^0.5.0",
+      "tough-cookie@<4.1.3": "^4.1.3",
+      "semver@>=7.0.0 <7.5.2": "^7.5.2",
+      "trim-newlines@<3.0.1": "^3.0.1",
+      "minimatch@>=3.0.0 <3.0.5": "^3.0.5",
+      "get-func-name@<2.0.1": "^2.0.1",
+      "postcss@<8.4.31": "^8.4.31"
```

Comment:
pnpm audit fix, targets only the broken versions and patches them on the minor available